components/navbar.tsx

```diff
@@ -1,18 +1,30 @@
-import React from "react";
+"use client";  // Mark this component as a Client Component
```

Comment:
remove this

Reply:
it will not work without this

i also try this  but it did not work

### const currentPath = typeof window !== "undefined" ? window.location.pathname : "";